google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/scaladsl/BigQueryProductFormats.scala

```diff
@@ -0,0 +1,1850 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.googlecloud.bigquery.scaladsl
+
+import spray.json.{
+  deserializationError,
+  DeserializationException,
+  JsArray,
+  JsObject,
+  JsValue,
+  JsonFormat,
+  JsonReader,
+  ProductFormats,
+  RootJsonFormat,
+  StandardFormats
+}
+
+import scala.reflect.{classTag, ClassTag}
+
+trait BigQueryProductFormats { this: ProductFormats with StandardFormats =>
```

Comment:
I think, generating the N formatter we have some generator syntax, I'm not really sure that this will work "as is" but akka-streams uses it [here](https://github.com/akka/akka/tree/master/akka-stream/src/main/boilerplate/akka/stream) and it makes less code.

Reply:
Cool, this was really easy to do with the sbt boilerplating plugin. Thanks!